SystemInfo/Exception/ComposerLockFileNotFoundException.php

```diff
@@ -15,6 +15,6 @@ class ComposerLockFileNotFoundException extends BaseNotFoundException
 {
     public function __construct($path, Exception $previous = null)
     {
-        parent::__construct('Composer lockfile', $path, $previous);
+        parent::__construct('Composer.lock file', $path, $previous);
```

Comment:
```suggestion
        parent::__construct('composer.lock file', $path, $previous);
```
If we're using the filename, it should be lower case.